repository_service_tuf/cli/admin/metadata.py

```diff
@@ -0,0 +1,520 @@
+        )
+        if not change:
+            console.print("Skipping further root metadata changes")
+            break
+
+        msg = "\nWhat should be the [cyan]root[/] role [green]threshold?[/]"
+        current_root.threshold = _get_positive_int_input(
+            msg, "threshold", current_root.threshold
+        )
+
+        console.print(markdown.Markdown(ROOT_KEYS_REMOVAL_MSG), width=100)
+        _keys_removal(current_root)
+
+        console.print(markdown.Markdown(ROOT_KEY_ADDITIONS_MSG), width=100)
+        _keys_additions(current_root)
+
+        console.print("\nHere is the current content of root:")
+
+        _print_curr_root(current_root)
+
+
+def _modify_online_key(current_root: RootInfo):
+    console.print(markdown.Markdown(ONLINE_KEY_CHANGE), width=100)
+    while True:
+        online_key_table = _create_keys_table(
+            current_root, is_online_key_table=True, is_minimal=False
+        )
+        console.print("\nHere is the information for the current online key:")
+        console.print("\n")
+        console.print(online_key_table)
+        console.print("\n")
+        change = prompt.Confirm.ask(
+            "Do you want to change the [cyan]online key[/]?"
+        )
+        if not change:
+            console.print("Skipping further online key changes")
+            break
+
+        online_key: RSTUFKey = _get_key(Root.type)
+        if online_key.error:
+            console.print(f":cross_mark: [red]Failed[/]: {online_key.error}")
+            continue
+
+        if online_key == current_root.online_key:
+            console.print(
+                ":cross_mark: [red]Failed[/]: The new online key is the same "
+                " as the current online key"
+            )
+            continue
+
+        if current_root.is_keyid_used(online_key.key["keyid"]):
+            console.print(
+                ":cross_mark: [red]Failed[/]: Key matches one of the root keys"
+            )
+            continue
+
+        online_key.name = prompt.Prompt.ask(
+            "[Optional] Give a [green]name/tag[/] to the key"
+        )
+
+        current_root.change_online_key(online_key)
+
+
+# ADD OPTION TO UPLOAD NEW ROOT CONTENT
+@metadata.command()
+@click.option(
+    "--current-root-uri",
+    help="URL or local path to the current root.json file.",
+    required=False,
+)
+@click.option(
+    "-f",
+    "--file",
+    "file",
+    default="metadata-update-payload.json",
+    help=(
+        "Generate specific JSON payload file"
+    ),
+    show_default=True,
+    required=False,
+)
+@click.pass_context
+def update(context, current_root_uri: str, file: str) -> None:
+    """
+    Start a new metadata update ceremony.
+    """
+    if current_root_uri is None:
+        current_root_uri = prompt.Prompt.ask(
+            "[cyan]File name or URL[/] to the current root metadata"
+        )
+    try:
+        curr_root: RootInfo = _get_curr_root(current_root_uri)
+    except StorageError:
```

Comment:
Do you want to add here `json.decoder.JSONDecodeError`?
It should be clear to the user that the file is incompatible.

See:
```shell
❯ rstuf admin metadata update
File name or URL to the current root metadata: Makefile
```
or
```shell
rstuf admin metadata update --current-root-uri Makefile
```

will fail like it
```shell
Traceback (most recent call last):
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/tuf/api/serialization/json.py", line 36, in deserialize
    json_dict = json.loads(raw_data.decode("utf-8"))
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/bin/rstuf", line 8, in <module>
    sys.exit(rstuf())
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/rich_click/rich_group.py", line 21, in main
    rv = super().main(*args, standalone_mode=False, **kwargs)
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/kdearaujo/devel/RSTUF/repository-service-tuf-cli/repository_service_tuf/cli/admin/metadata.py", line 488, in update
    curr_root: RootInfo = _get_curr_root(current_root_uri)
  File "/Users/kdearaujo/devel/RSTUF/repository-service-tuf-cli/repository_service_tuf/cli/admin/metadata.py", line 115, in _get_curr_root
    curr_root_md = Metadata.from_file(curr_root_uri)
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/tuf/api/metadata.py", line 234, in from_file
    return cls.from_bytes(file_obj.read(), deserializer)
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/tuf/api/metadata.py", line 264, in from_bytes
    return deserializer.deserialize(data)
  File "/Users/kdearaujo/.local/share/virtualenvs/repository-service-tuf-cli-fl9agVA1/lib/python3.10/site-packages/tuf/api/serialization/json.py", line 40, in deserialize
    raise DeserializationError("Failed to deserialize JSON") from e
tuf.api.serialization.DeserializationError: Failed to deserialize JSON
```